### ISSUE TITLE
Download not working in webapp clicks

### ISSUE THREAD

**Issue by MK1973** [ubports/morph-browser#209]
**Description of the feature**

    Device: N5, Bq 4.5, Fairphone, I think all
    Channel: Stable, RC, Devel
    Build: OTA 7, 8, 9

**Expected behavior**

- when clicking to download file at web-app, will open contenhub to select where to download and after that the file will be downloaded

**Actual behavior**

- open any web app, click on download and nothing happens, no changes, not downloading, loading not opening any content hub window

- opening the same web page directly with Morph without a web app it works well

**Logfiles and additional information**

- no logs to report
- do not know if it is only an issue due to created web app by clickable app or also using clickable desktop tool

**Comment by Stefano0101** [comment]
Also not working on Nexus 7 and Meizu Pro 5

**Comment by balcy** [comment]
yes it is currently not working, I've prepared a branch for it.
For opening the downloaded files with other apps (e.g. filemanager) the webapps will need the content_exchange permission.

**Comment by josele13** [comment]
I have done many webapp, some are access to emails,
these web apps have various permissions enabled, including -- content_exchange and content_exchange_source,

none of them download files 

after giving some permission:
" sudo mount -t tmpfs tmpfs /var/cache/apt/archives"

I have done the installation by terminal:
"sudo ubports-qa install xenial_-_downloadswebapps"

After installation, I have restarted the mobile,

I have tried several webapps to see if they can download files, the result is that they open the Morph browser or do not act.

should open the download manager and doesn't.

this installation does not work properly.
